src/shared/components/saveButton/saveButton.component.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FormattedMessage } from 'react-intl';
+
+import messages from './saveButton.messages';
+import { Button } from './saveButton.styles';
+
+const SaveButton = ({ saveResult }) => {
+  const handleClick = () => saveResult();
+
+  return (
+    <Button type="button" onClick={handleClick}>
```

Comment:
```suggestion
    <Button type="button" onClick={saveResult}>
```
Referencja tutaj nie powinna się zmienić więc nie ma sensu robienie wrappera

Reply:
Eslint wymusił na mnie metodę o nazwie handle